apps/docs/docs/how-to-guides/platform-integration/vtex/setting-up-an-account.md

```diff
@@ -23,26 +23,14 @@ Before proceeding any further, make sure you have:
 
 ## Step by step
 
-In this step by step, you'll use the FastStore plugin for the VTEX IO CLI to install, with a single command, all dependencies needed for developing a FastStore website.
+In this step by step, you'll install the FastStore Admin app in your account via the VTEX IO CLI.
 
-1. Install the FastStore plugin for the VTEX IO CLI:
+1. Install the FastStore Admin app in your account:
 
    ```
-   vtex plugins install faststore
+   vtex install vtex.admin-faststore
    ```
 
-2. Log in to your VTEX account:
+2. Open the Admin app at `https://{workspace}--{account}.myvtex.com/admin/faststore` and follow the setup steps.
```

Comment:
Are we recommending using a workspace (other than master) on this setup?
I only worry that people start to think that they'll need to use this specific workspace when using faststore